lib/memjs/memjs.js

```diff
@@ -401,6 +402,35 @@ Client.prototype.stats = function(callback) {
   }
 }
 
+
+// STATS
+//
+// Fetches memcache stats from each connected server. The callback is invoked 
+// **ONCE PER SERVER** and has the signature:
+//
+//     callback(err, server, stats)
+//
+// _server_ is the `"hostname:port"` of the server, and _stats_ is a
+// dictionary mapping the stat name to the value of the statistic as a string.
+Client.prototype.stats = function(callback) {
+  this.statsWithKey('', callback);
+}
+
+// RESET_STATS
+//
+// Reset the statistics each server is keeping back to zero. This doesn't clear
+// stats such as item count, but temporary stats such as total number of
+// connections over time.T
+//
+// The callback is invoked **ONCE PER SERVER** and has the signature:
+//
+//     callback(err, server)
+//
+// _server_ is the `"hostname:port"` of the server.
+Client.prototype.resetStats = function(callback) {
```

Comment:
I disagree. My reasoning:

1) Dalli does it this way, it has `reset_stats` and `stats(key=nil)`.
2) `resetStats` is a common operation, and it feels obtuse to me to hide it as a string passed to `statsWithKey`, the underlying protocol may do this but as an API it feels odd and hard to find. The user would have to look at the comments and not function names.
3) All the other key options for `statsWithKey` are extended stats (i.e., `statsWithKey('items')`) and in REST terms, are GET requests. While `'reset'` is a modifying request (DELETE / PUT). Feels right to separate those.

Thoughts? If you still like your alternative (and you know the spirit of the library while I dont), then say so and I'll make the change.